### ISSUE TITLE
Volume buttons match labels and not command

### ISSUE THREAD

**Issue by lynchhj1** [bwssytems/ha-bridge#33]
While tinkering in the latest version (1.3.5) i added harmony device button presses for volume/previous channel and they did not work.  After looking up commands i was able to manually change and they worked correctly.

**Comment by bwssytems** [comment]
A Question, did you use the builder that is now there in 1.3.5 to configure the buttons? You can set and then click build and it will add each selection to the device config. When you are done, add the device.

**Comment by audiofreak9** [comment]
It seems I ran into the same issue with volume, my new device "sound" was created with the following: 

ON: [{"device":"32829111","button":"Volume Up"},{"device":"32829111","button":"Volume Up"}]
OFF: [{"device":"32829111","button":"Volume Down"},{"device":"32829111","button":"Volume Down"}]

The Echo responds, "The device sound is not responding...", and the Test On/Off buttons fail on the configuration page.  All my other Harmony buttons and activities work.

I removed the space between "Volume" and "Up" & "Down" for the ON & OFF and they worked, here is my manually revised:

ON: [{"device":"32829111","button":"VolumeUp"},{"device":"32829111","button":"VolumeUp"}]
OFF: [{"device":"32829111","button":"VolumeDown"},{"device":"32829111","button":"VolumeDown"}]

**Comment by bwssytems** [comment]
That is not good. For numbers the label needs to be used and not the name. Will need more research then.
Thanks
On Jan 28, 2016 9:50 PM, audiofreak9 <notifications@github.com> wrote:It seems I ran into the same issue with volume, my new device "sound" was created with the following: 

ON: [{"device":"32829111","button":"Volume Up"},{"device":"32829111","button":"Volume Up"}]
OFF: [{"device":"32829111","button":"Volume Down"},{"device":"32829111","button":"Volume Down"}]

The Echo responds, "The device sound is not responding...", and the Test On/Off buttons fail on the configuration page.  All my other Harmony buttons and activities work.

I removed the space between "Volume" and "Up" & "Down" for the ON & OFF and they worked, here is my manually revised:

ON: [{"device":"32829111","button":"VolumeUp"},{"device":"32829111","button":"VolumeUp"}]
OFF: [{"device":"32829111","button":"VolumeDown"},{"device":"32829111","button":"VolumeDown"}]
